src/Components/InstanceTypesSelect/InstanceTypesSelect.test.js

```diff
@@ -47,7 +47,10 @@ describe('InstanceTypesSelect', () => {
 });
 
 const mountSelectAndClick = async (provider = 'aws') => {
-  render(<InstanceTypesSelect architecture="x86_64" setValidation={jest.fn()} />, { provider, contextValues: { chosenSource: '1' } });
+  render(<InstanceTypesSelect architecture="x86_64" setValidation={jest.fn()} setvCPUCount={jest.fn()} />, {
```

Comment:
I do not insist, but you could save the function to a var and test whether it gets called with a correct value.